libraries/chain/include/graphene/chain/asset_object.hpp

```diff
@@ -65,6 +65,7 @@ namespace graphene { namespace chain {
          share_type current_supply;
          share_type confidential_supply; ///< total asset held in confidential balances
          share_type accumulated_fees; ///< fees accumulate to be paid out over time
+         share_type accumulated_collateral_fees; ///< accumulated collateral-denominated fees (for bitassets)
```

Comment:
This new field need to be reflected.

Reply:
Ah, thanks for this.  Didn’t see where these fields were reflected at first as the reflection is apparently in the cpp instead of the hpp.  Fixed in 02f9da8.